packages/fast-web-utilities/src/dom.spec.ts

```diff
@@ -36,3 +36,9 @@ describe("getKeyCode", () => {
         expect(getKeyCode(new KeyboardEvent("keypress", {charCode: 9} as KeyboardEventInit))).toBe(KeyCodes.tab);
     });
 });
+
+describe("isDescendant", () => {
+    test("should correctly handle null", () => {
```

Comment:
This does not seem like adequate testing. Please add in sufficient tests for:
- testing an immediate child
- testing for a child further down the tree of a component
- testing for a adjacent element to the one being checked (false)
- testing for a parent being the descendent of its child (false)

Reply:
redundant, see @chrisdholt comments

Reply:
Well this was the testing we provided for it in our fluent repo. May be moot now anyway.